platform/features/conductor/src/ConductorTelemetrySeries.js

```diff
@@ -0,0 +1,71 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*global define*/
+
+define(
+    function () {
+        'use strict';
+
+
+        /**
+         * Bound a series of telemetry such that it only includes
+         * points from within the time conductor's displayable window.
+         *
+         * @param {TelemetrySeries} series the telemetry series
+         * @param {platform/features/conductor.TimeConductor} the
+         *        time conductor instance which bounds this series
+         * @constructor
+         * @implements {TelemetrySeries}
+         */
+        function ConductorTelemetrySeries(series, conductor) {
```

Comment:
This entire wrapper seems unnecessary-- the underlying telemetry providers should not be providing data that is outside of the requested `start` and `end` dates.  If they do that, then the binary search that occurs here is worst case performance, every time.  Also forces all domainValues to be a date.

Also has the side effect of forcing the time conductor functionality onto all components-- no matter what the start date and end date were in the request, the time conductor will force the responses to exclude data outside that range.